<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>窗口调整工具</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>窗口列表(标题--句柄)</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lwWindows">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>411</width>
     <height>661</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRefresh">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>10</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>85</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbPosX">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>-1920</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="title">
    <string>窗口左上角坐标</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbPosY">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>125</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>-1080</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>185</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="title">
    <string>窗口尺寸</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbSizeX">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>215</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>-1920</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>265</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbSizeY">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>-1080</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbOffsetY">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>395</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>-1080</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="title">
    <string>窗口间位置偏移</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>400</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>355</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbOffsetX">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>-1920</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbFirstToTop">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>480</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>首窗口置顶</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbLastToTop">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>520</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>末窗口置顶</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>450</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="title">
    <string>窗口排序</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbPos">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>580</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>调整位置</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbSize">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>610</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>调整尺寸</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOffset">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>640</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>使用偏移</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExecute">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>670</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确 定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>720</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>---- by tenting</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>720</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>不调整位置,只调整尺寸,使用偏移会失效</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>lwWindows</zorder>
  <zorder>btnRefresh</zorder>
  <zorder>groupBox</zorder>
  <zorder>sbPosX</zorder>
  <zorder>label_3</zorder>
  <zorder>sbPosY</zorder>
  <zorder>label_4</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>sbSizeX</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>sbSizeY</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>sbOffsetX</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>rbLastToTop</zorder>
  <zorder>rbFirstToTop</zorder>
  <zorder>cbPos</zorder>
  <zorder>cbSize</zorder>
  <zorder>cbOffset</zorder>
  <zorder>btnExecute</zorder>
  <zorder>sbOffsetY</zorder>
  <zorder>label_2</zorder>
  <zorder>label_9</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
